sa/sa.go

```diff
@@ -216,13 +216,15 @@ func (ssa *SQLStorageAuthority) GetAuthorization(ctx context.Context, id string)
 	if err != nil {
 		return authz, err
 	}
-	pa, err := selectPendingAuthz(tx, "WHERE id = ?", id)
+	txCtx := tx.WithContext(ctx)
```

Comment:
nit: this naming scheme is slightly confusing as almost everywhere else we use `xxxCtx` to refer to a context object we are using for a specific purpose, not a thing using a context. That said I can't really come up with something better.